### PR TITLE
Installation verification procedure for Ollama 

### PR DESCRIPTION
**Pull request by Badhansen**
- [x] Make sure `Ollama` up and running the machine.
- [x] Ensure the `Embedder` and `Generator` have been selected before going ahead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,8 +149,21 @@ Verba supports Ollama models. Download and Install Ollama on your device (https:
 
 Tested with `llama3`, `llama3:70b` and `mistral`. The bigger models generally perform better, but need more computational power.
 
+> Ensure that you have the right configurations for the `Embedder` and `Generator` selected before going ahead.
+
+![verba-embedder](https://github.com/Badhansen/Verba/assets/37157382/4aedceed-e5e2-4e72-9273-bcd1ccc6f471)
```
Comment (thomashacker) [inline README.md:154]: Can we add the images to the repo as well?

Comment (Badhansen) [reply README.md:154]: @thomashacker Sure! I would be delighted to help with this project.

Comment (thomashacker) [reply README.md:154]: Could you add the images to the PR and adjust the links? Thanks

Comment (Badhansen) [reply README.md]: Hi @thomashacker I updated the links and added the images. Thank you.

**Review by thomashacker — APPROVED** [review]
Looks good to me, thanks a lot!

**Pull request merged**